package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@codahq/packs-sdk",
-  "version": "1.7.7",
+  "version": "1.7.8",
```

Comment:
I don't think you should be bumping the SDK version, or if you do it should be to a pre-release version.

Reply:
@patrick-codaio Are we able to use packs on head w/ prerelease versions?

Reply:
I'm fairly certain the answer is yes, we've done that before.  The issue is with using non-released versions.  And the only issue that I know of with that is that lint starts taking ~3x as long (~an hour).

Reply:
I thought we made a change where we now use the Live version of packs vs Latest which I thought had to be on a real release?

Reply:
There's a flag that controls whether Coda Brain uses Live or Latest pack versions, and it now defaults to Live everywhere (also you can't Release a pack version built with a pre-release SDK version). 

Are you sure you need this running on head now, and not just your dev? 

Releasing a new version isn't terrible, though it is a bit of a chore - https://staging.coda.io/d/Packs-Platform-IA-go-packs_d36WK4zgrYx/Releasing-Packs-SDK-Versions-o-release-packs-sdk_suerz#_luCH6

Reply:
I can start w/ dev for now and then move to head next week